### ISSUE TITLE
System.IO.FileNotFoundException

### ISSUE THREAD

**Issue by Steviey** [xamarinhq/app-acquaint#18]
Severity    Code    Description Project File    Line    Suppression State
Error       Unexpected error - Please file a bug report at http://bugzilla.xamarin.com. Reason: System.IO.FileNotFoundException: Could not load assembly 'Acquaint.Abstractions, Version=, Culture=neutral, PublicKeyToken='. Perhaps it doesn't exist in the Mono for Android profile? Acquaint.Native.Droid

**Comment by karlbuys** [comment]
I can confirm that I experienced the same issue, when attempting to build the Native Acquaint solution.

Steps to replicate this issue (for me) in VS 2015 Update 3 are as follows:

1. Double-click Acquaint.Native.sln
2. Build
3. Error message stipulating that Android.Abstractions.dll could not be found in "\App\Common\Acquaint.Abstractions\bin\Debug".

I managed to get this .dll file generated by opening Acquaint.XForms.sln, and then building that solution.

I'm an absolute Visual Studio 2015 newbie, but it seems to me that there's some directive missing that would instruct the Native solution to build successfully right from the outset.

Hope this solves the issue for someone else. If anyone has an idea why this issue occurs, I'd love to know the reason why and the difference between the two projects for personal growth.
